src/test/java/com/fasterxml/jackson/databind/node/JsonNodeFactoryTest.java

```diff
@@ -80,15 +80,20 @@ public void testSortingObjectNode() throws Exception
                MAPPER.writeValueAsString(mapper.readTree(BIGGER_INPUT)));
    }
 
-   // 06-Nov-2022, tatu: Wasn't being tested, oddly enough
-   public void testBigDecimalNormalization() throws Exception
+	public void testBigDecimalNormalization_enabled_by_default() throws Exception
+	{
+		final BigDecimal NON_NORMALIZED = new BigDecimal("12.5000");
```

Comment:
please fix the tabs - use spaces not tabs

Reply:
@pjfanning oops, got it

Reply:
thanks - seems like the indentation in this class is a bit erratic - a mix of 3 space and 4 space indents

Reply:
word, I just followed the existing code convention.

Reply:
@pjfanning btw, are explicit (unrequired) type-casting intentional?

If not, I can try to work on below lint warnings from **sonarlint** on `JsonNodeFactory` for a start
 
![image](https://user-images.githubusercontent.com/61615301/215270580-11df3ade-148a-4c1f-9752-835d60ade4fe.png)

Reply:
best to do any refactors like that in a separate PR

Reply:
@pjfanning Well understood. This PR was intended only for the test separation only.